app/Server.php

```diff
@@ -265,7 +356,10 @@ public function updateUsage()
                 $this->timestamps              = false;
                 $this->save();
 
-                $this->stats()->save($serverStat);
```

Comment:
Shouldn't the live server status (see the lines above) also be empty is the server statistics disabled?

Reply:
No, we need the live status for load balancing

Reply:
The option should only disable archiving this data